.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
+logs/
```

Comment:
Ensure that the `logs/` directory is correctly ignored. This is important for preventing log files from being committed to the repository, which is a common practice to keep the repository clean and secure.